src/main/java/io/confluent/examples/streams/PageViewRegionExampleDriver.java

```diff
@@ -82,21 +81,20 @@ private static void produceInputs(String bootstrapServers, String schemaRegistry
 
     final Random random = new Random();
     pageViewBuilder.set("industry", "eng");
-    for (final String user : users) {
-      userProfileBuilder.set("experience", "some");
-      userProfileBuilder.set("region", regions[random.nextInt(regions.length)]);
-      producer.send(new ProducerRecord<>(userProfilesTopic, user, userProfileBuilder.build()));
-      // For each user generate some page views
-      IntStream.range(0, random.nextInt(10))
-        .mapToObj(value -> {
+    try (final KafkaProducer<String, GenericRecord> producer = new KafkaProducer<>(props)) {
+      for (final String user : users) {
+        userProfileBuilder.set("experience", "some");
+        userProfileBuilder.set("region", regions[random.nextInt(regions.length)]);
+        producer.send(new ProducerRecord<>(userProfilesTopic, user, userProfileBuilder.build()));
+        // For each user generate some page views
+        IntStream.range(0, random.nextInt(10)).forEach(value -> {
```

Comment:
I am not a Scale expert. Why do you remove `mapToObj` ?

Reply:
For better readability, in this case, we don't need to do two operations.
`mapToObj` combined with `forEach` is same as the above one.

Reply:
Ack. Thanks for clarifying